spring-cloud-dataflow-server/docker-compose.yml

```diff
@@ -11,21 +11,21 @@ services:
     expose:
       - 3306
 
-  kafka:
-    image: confluentinc/cp-kafka:5.2.1
+  kafka-broker:
```

Comment:
Is there any specific reason to change this name to `kafka-broker` ?

Reply:
I guess it is just convenient.

Reply:
No, unfortunately this is an enforced requirement due to  https://github.com/helm/charts/issues/5089 and https://github.com/helm/charts/pull/5104/files#diff-96171ed0f9aefa1650847d0c34f3ed60R175

If you try to use `kafka` as a name, the official kafka docker images you fail with an error like this: `port is deprecated. Please use KAFKA_ADVERTISED_LISTENERS instead.`

While the error message is misleading the core of the issue is that an implicit env. variable is created with name `<Your-Kafka-Service-Name>_PORT`.  E.g. for `kafka` name a `KAFKA_PORT` variable is created. But the `KAFKA_PORT` is listed as deprecated and the image script aborts.

There is an open issue to relax the failure on deprecated variables: https://github.com/confluentinc/cp-docker-images/issues/388

But until it is resolved we can not use `kafka` as a name.  Btw i've been using `dataflow-kafka` in our the docker-compose files.